lib/quandl/api_config.rb

```diff
@@ -30,7 +30,7 @@ def api_base
       end
 
       def api_version=(api_version)
-        @api_version = api_base if @api_version.nil?
+        @api_version ||= api_base
```

Comment:
@iamjarvo could you also correct the variable name which seems to wrong here also?

Reply:
Good catch! That was a silly mistake on my part. I tried to add a test for that error but the suite still passed. At some point I can take a look at that